Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportCsvMapper.cs

```diff
@@ -59,6 +60,11 @@ private static string MapBoolean(bool input)
             return input ? Yes : No;
         }
 
+        private static string MapDateTime(DateTime? input)
+        {
+            return input == null ? "" : input.GetValueOrDefault().ToShortDateString();
```

Comment:
does this render like the other fields in kitos: date-month-year

Reply:
no, 
![image](https://user-images.githubusercontent.com/13116889/206160949-dad76148-7625-4dfb-b28b-7b36c169f2c7.png)

Reply:
in order to make sure, you should specify the format.
The one above works on the right server locale, but specifying the format will allow you to **guarantee** the format